docs/satellite-clone.md

```diff
@@ -14,8 +14,11 @@
 #### Prerequisites
 
 1. You will need files from a katello-backup (`katello-backup` on the `Satellite server`).
-   - The playbook will work with or without the pulp_data.tar file from the backup.
-   - *Note:* For cloning a rhel6 Satellite server to a rhel7 machine: In addition to katello-backup created files, you also need foreman.dump and candlepin.dump.  You may utilize the included script [satellite-clone/helpers/postgresql_dump.sh)(../helpers/postgresql_dump.sh) if needed.
+   Required backup files:
+   - Standard backup scenario : config_files.tar.gz, mongo_data.tar.gz, pgsql_data.tar.gz, (optional) pulp_data.tar
+   - Online backup scenario   : config_files.tar.gz, mongo_dump folder, foreman.dump, candlepin.dump
+   - rhel migration scenario  : config_files.tar.gz, mongo_data.tar.gz, foreman.dump, candlepin.dump, (optional) pulp_data.tar
+     For rhel migration scenario, you may utilize the included script [satellite-clone/helpers/postgresql_dump.sh)(../helpers/postgresql_dump.sh) if needed.
```

Comment:
Thank you for incorporating my suggestions here. 

Now the pre_install_check should make sure to check for the presence of the required backup files in each of these scenarios.